hackathon_site/dashboard/frontend/src/components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable.tsx

```diff
@@ -0,0 +1,280 @@
+                                    <GeneralOrderTableTitle
+                                        orderId={pendingOrder.id}
+                                        orderStatus={pendingOrder.status}
+                                    />
+                                    <TableContainer
+                                        component={Paper}
+                                        elevation={2}
+                                        square={true}
+                                    >
+                                        <Table className={styles.table} size="small">
+                                            <TableHead>
+                                                <TableRow>
+                                                    <TableCell
+                                                        className={styles.widthFixed}
+                                                    />
+                                                    <TableCell
+                                                        className={styles.width6}
+                                                    >
+                                                        Name
+                                                    </TableCell>
+                                                    <TableCell
+                                                        className={`${styles.width1} ${styles.noWrap}`}
+                                                    >
+                                                        Model
+                                                    </TableCell>
+                                                    <TableCell
+                                                        className={`${styles.width1} ${styles.noWrap}`}
+                                                    >
+                                                        Manufacturer
+                                                    </TableCell>
+                                                    <TableCell
+                                                        className={`${styles.width1} ${styles.noWrap}`}
+                                                    >
+                                                        Qty requested
+                                                    </TableCell>
+                                                    <TableCell
+                                                        className={`${styles.width6} ${styles.noWrap}`}
+                                                    >
+                                                        Qty granted
+                                                    </TableCell>
+                                                    <TableCell
+                                                        className={`${styles.width1} ${styles.noWrap}`}
+                                                    >
+                                                        <Checkbox color="primary" />
```

Comment:
could you add a setFieldValue call here to check all the boxes: 
![image](https://user-images.githubusercontent.com/20618460/180909574-458d6a6a-b051-4f93-bf0c-35ff4ea85af0.png)

you should also write a test for this because it's custom functionality